crates/matrix-sdk-crypto/src/lib.rs

```diff
@@ -112,3 +113,25 @@ matrix_sdk_test::init_tracing_for_tests!();
 
 #[cfg(feature = "uniffi")]
 uniffi::setup_scaffolding!();
+
+/// The trust level in the sender's device that is required to decrypt an
+/// event. If the sender's device is not sufficiently trusted,
+/// [`MegolmError::SenderIdentityNotTrusted`] will be returned.
```

Comment:
Nit: this second sentence maybe applies more to `DecryptionSrettings::sender_device_trust_requirement` than the type.